scripts/generate-patch

```diff
@@ -58,16 +58,41 @@ for f in packages/*; do
 				mkdir -p ${f}/charts-original
 				curl -sLf ${url} | tar xvzf - -C ${f}/charts-original --strip ${fields} ${subdirectory} > /dev/null 2>&1
 			fi
+			if [[ -f ${f}/charts/requirements.yaml ]]; then
+				i=0
+				while ! [[ -z $(yq r ${f}/charts/requirements.yaml "dependencies[${i}]") ]]; do
+					subchart=$(yq r ${f}/charts/requirements.yaml "dependencies[${i}].name" | head -n 1)
+					version=$(yq r ${f}/charts/requirements.yaml "dependencies[${i}].version" | head -n 1)
+					helm_repo=$(yq r ${f}/charts/requirements.yaml "dependencies[${i}].repository" | head -n 1)
```

Comment:
is there ever a time when a repository is not listed on a dependency?

Reply:
@brendarearden this particular change is actually introduced in https://github.com/rancher/charts/pull/579, not part of this PR.

Reply:
> is there ever a time when a repository is not listed on a dependency?

Just double checked tho; these are guaranteed by Helm based on the [docs](https://helm.sh/docs/topics/charts/#chart-dependencies)